app/controllers/services_controller.rb

```diff
@@ -0,0 +1,6 @@
+class ServicesController < ApplicationController
+  def register
+    @service = Service.create
+    render json: @service, include: :models
```

Comment:
I think duplicate model names should not be a "failure" though.

As we discussed @dfaulken, the engine initializers will send a request to the router on every time it is started using the information in the YML file generated by the engine.

If the model name is equal, it should check the endpoint for it. If they're both the same, just ignore it. If the endpoint for the model changes, then it should update the endpoint instead of just calling it a failure, right? 

The engine won't know the difference between a first-time startup and the hundredth startup, so it's not like we can just have it send to a different endpoint on the router based on that.